lib/core/include/irods/rodsErrorTable.h

```diff
@@ -318,6 +318,8 @@ NEW_ERROR(USER_INCOMPATIBLE_OPEN_FLAGS,                -404000)
 NEW_ERROR(INTERMEDIATE_REPLICA_ACCESS,                 -405000)
 NEW_ERROR(LOCKED_DATA_OBJECT_ACCESS,                   -406000)
 NEW_ERROR(CHECK_VERIFICATION_RESULTS,                  -407000)
+NEW_ERROR(GENQUERY2_END_OF_RESULTSET,                  -408000)
```

Comment:
New error code for GenQuery2 MSI.

Reply:
Perhaps this should be called `END_OF_RESULTS`.

That's more generic and can be used for other things needing a stop code.

Reply:
that seems reasonable

Reply:
Done.